apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -28,6 +30,15 @@ mod device;
 mod port;
 
 const APP_NAME: &str = "vx-mark-scan-controller-daemon";
+const POSSIBLE_DEVICE_PATHS: [&str; 2] = ["/dev/ttyACM0", "/dev/ttyACM1"];
```

Comment:
I went back and forth on using `available_ports` to list ports rather than just hardcode. I'll try out `available_ports` on hardware at next availability and possibly update this.

Reply:
https://github.com/votingworks/vxsuite/pull/4695/commits/454ff9f6d85379e1317715dbadcd666586cb24f0